pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -501,6 +503,12 @@ func mutateContainerResource(c *corev1.Container, override *v1beta1.OverrideSpec
 			}
 		}
 	}
+
+	if loglevel, ok := override.LogLevel[c.Name]; ok {
+		c.Args = append(c.Args, fmt.Sprintf("--v=%d", loglevel))
+	} else {
+		c.Args = append(c.Args, fmt.Sprintf("--v=%d", defaultLogLevel))
```

Comment:
It looks like this changes the default of the `git-sync` container, as it was defaulting to `5` before

Reply:
updated to make an exception for git-sync container